ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -92,15 +105,54 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
 		containerDefinitions = append(containerDefinitions, containerDef)
 	}
 
+	ecsParams, err := readECSParams(ecsParamsFileName)
+	if err != nil {
+		return nil, err
+	}
+
+	var networkMode string
+	if ecsParams != nil {
+		networkMode = ecsParams.TaskDefinition.NetworkMode
+		// The task-role-arn flag should take precedence over a taskRoleArn value specified in ECS fields file
+		if taskRoleArn == "" {
+			taskRoleArn = ecsParams.TaskDefinition.TaskRoleArn
+		}
+	}
+
 	taskDefinition := &ecs.TaskDefinition{
 		Family:               aws.String(taskDefinitionName),
 		ContainerDefinitions: containerDefinitions,
 		Volumes:              convertToECSVolumes(volumes),
 		TaskRoleArn:          aws.String(taskRoleArn),
+		NetworkMode:          aws.String(networkMode),
 	}
+
 	return taskDefinition, nil
 }
 
+func readECSParams(filename string) (*ECSParams, error) {
+	if filename == "" {
+		defaultFilename := "ecs-params.yml"
+		if _, err := os.Stat(defaultFilename); err == nil {
```

Comment:
Should we also do this check when someone specifies a filename before reading the file?